package.json

```diff
@@ -2,11 +2,18 @@
   "name": "operations-orchestration-api",
   "version": "1.1.3",
   "description": "HPE's Operations Orchestration (OO) client API module for NodeJS",
-  "main": "index.js",
+  "main": "./dist/index.js",
   "scripts": {
     "test": "mocha test/test.js",
```

Comment:
@Attrash-Islam because we are now transpiling the package how about we update the test run-script to:
`"test": "npm run build && mocha test/test.js"`